scripts/azure-windows.yml

```diff
@@ -26,21 +55,36 @@ steps:
     # currently we do not run S3 tests on Windows because tests time out (minio+azure slow?)
     #& "$env:BUILD_SOURCESDIRECTORY\scripts\install-minio.ps1"
 
+    $bootstrapOptions = "-EnableVerbose -EnableStaticTileDB -EnableBuildDeps"
     if ($env:TILEDB_S3 -eq "ON") {
-      & "$env:BUILD_SOURCESDIRECTORY\bootstrap.ps1" -EnableS3 -EnableVerbose -EnableStaticTileDB -EnableBuildDeps
-    } else {
-      & "$env:BUILD_SOURCESDIRECTORY\bootstrap.ps1" -EnableVerbose -EnableStaticTileDB -EnableBuildDeps
+      $bootstrapOptions = "-EnableS3 " + $bootstrapOptions
+    }
+    if ($env:TILEDB_AZURE -eq "ON") {
+      #delay install/run of azurite until actually needed before 'make
+      #check' rather than unnecessarily consuming time/resources it takes
+      #downloading/expanding/installing node/azurite since other failures may occur prior.
+      #& "$env:BUILD_SOURCESDIRECTORY\scripts\install-run-azurite.ps1"
```

Comment:
Is this a legitimate comment or dead code?

Reply:
Both.  But not necessary. Removed.